Library/OcDebugLogLib/OcLog.c

```diff
@@ -685,11 +647,19 @@ OcConfigureLogProtocol (
       //
       // Write filters into Private.
       //
-      FlexFilters = NULL;
-      if (AsciiStrCmp (LogModules, "*") != 0 && AsciiStrCmp (LogModules, "") != 0) {
-        FlexFilters = OcStringSplit (LogModules, L',', FALSE);
+      Private->FlexFilters = NULL;
+      Private->FilterMode = OcLogFilterModeNoFilter;
```

Comment:
Looks not correct, because NoFilter is invalid, and there should be no branch for no filter. In my opinion, make FilterMode a boolean with a more descriptive name: `IsBlacklistFilter` or something.

Reply:
https://github.com/acidanthera/OpenCorePkg/pull/322/commits/8a869c21e0819e132581bc7e4ec66a3b87ec70d0
https://github.com/acidanthera/OpenCorePkg/pull/322/commits/c21c78fa91011f1dad5c3857964a999f3e1f29df